commerce.views.inc

```diff
@@ -0,0 +1,16 @@
+<?php
+
+/**
+ * Implements hook_views_data_alter().
+ */
+function commerce_views_data_alter(array &$data) {
+  // Override the default bundle views handler for all content entity types.
+  // Get all entity types.
+  $entity_types = \Drupal::service('entity_type.manager')->getDefinitions();
+  foreach ($entity_types as $entity_type) {
+    // We only want content entities.
```

Comment:
Careful, we should limit ourselves to commerce content entities only (id starts with "commerce_").